CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- `gateway-graphql` version
```

Comment:
To what version?

Reply:
Major version.

Reply:
I know, I'm asking you to add this information to the changelog. It is good to know that the version changed, but it is much better to know what version it was changed to.